src/sentry/models/auditlogentry.py

```diff
@@ -280,6 +280,8 @@ def get_note(self):
         elif self.event == AuditLogEntryEvent.PROJECT_EDIT:
             return "edited project settings " + (
                 " ".join(f" in {key} to {value}" for (key, value) in self.data.items())
+                if self.data["old_slug"] is None
+                else "in " + self.data["old_slug"] + " to " + self.data["new_slug"]
```

Comment:
is it possible to edit multiple settings at once? does this need to be in the loop? also, should it say something to the effect of
>changed project setting "slug" from "old_slug" to "new_slug"